modules/policy-change-management.adoc

```diff
@@ -33,6 +33,21 @@ An authorized SRE reviewer must approve advancement to each step. The reviewer c
 
 Some changes are released to production incrementally, using feature flags to control availability of new features to specified clusters or customers.
 
+You can implement a maintenance window by avoiding changes in {cluster-manager} for the following components:
+
+* Creating a cluster
+* Deleting a cluster
+* Adding or removing an identity provider
+* Adding or removing a user from an elevated group
+* Configuring cluster privacy
+* Adding or removing machine pools and configure autoscaling
+* Defining upgrade policies
+
+[IMPORTANT]
+====
+To implement a maintenance window, make sure to turn off auto scaling. After the maintenance has completed, turn auto scaling back on.
```

Comment:
```suggestion
To enforce a maintenance window, ensure autoscaling or automatic upgrade policies have been disabled. After the maintenance has completed, proceed with enabling autoscaling or automatic upgrade policies as desired.
```

Reply:
Updated in new commit with @billmvt suggestion from below as well.